### ISSUE TITLE
Support Rails 5

### ISSUE THREAD

**Issue by seanlinsley** [activeadmin/activeadmin#4177]
#4175 got Rails 5 (master) running minimally on Travis. Currently it's breaking while building the test app, due to problems with our dependencies.

The Draper issue described on #4175 can be fixed by adding this line:
```ruby
gem 'activemodel-serializers-xml', github: 'rails/activemodel-serializers-xml'
```

After that, I ran into this error, because Rails 5 automatically includes turbolinks in the `application.js` manifest file:
```
spec/rails/rails-5.0.0.alpha/app/assets/config/manifest.js:3: couldn't find file 'turbolinks' with type 'application/javascript' (Sprockets::FileNotFound)
  from spec/rails/rails-5.0.0.alpha/app/assets/javascripts/application.js:15
  from bundler/gems/sprockets-5a77f8b007b8/lib/sprockets/resolve.rb:58:in `resolve!'
```

Which can be fixed by adding it to the gemfile:
```ruby
gem 'turbolinks'
```

Rerunning, I came across this issue with Devise:
```
gems/devise-3.5.2/lib/devise/failure_app.rb:9:in `<class:FailureApp>': uninitialized constant ActionController::RackDelegation (NameError)
  from gems/devise-3.5.2/lib/devise/failure_app.rb:8:in `<module:Devise>'
  from gems/devise-3.5.2/lib/devise/failure_app.rb:3:in `<top (required)>'
  from gems/devise-3.5.2/lib/devise/mapping.rb:122:in `default_failure_app'
  from gems/devise-3.5.2/lib/devise/mapping.rb:67:in `initialize'
  from gems/devise-3.5.2/lib/devise.rb:321:in `new'
```

It looks like Devise has a fairly large outstanding PR to support Rails 5: https://github.com/plataformatec/devise/pull/3714, so we can use that branch for now:
```ruby
gem 'devise', github: 'twalpole/devise', branch: 'rails5'
```

I then ran into this issue:
```
        rake    db:migrate db:test:prepare
rake aborted!
LoadError: cannot load such file -- rails/test_unit/sub_test_task
bundler/gems/rails-a8876760041f/activesupport/lib/active_support/dependencies.rb:302:in `require'
bundler/gems/rails-a8876760041f/activesupport/lib/active_support/dependencies.rb:302:in `block in require'
bundler/gems/rails-a8876760041f/activesupport/lib/active_support/dependencies.rb:268:in `load_dependency'
bundler/gems/rails-a8876760041f/activesupport/lib/active_support/dependencies.rb:302:in `require'
gems/draper-2.1.0/lib/draper/tasks/test.rake:7:in `<top (required)>'
bundler/gems/rails-a8876760041f/activesupport/lib/active_support/dependencies.rb:296:in `load'
bundler/gems/rails-a8876760041f/activesupport/lib/active_support/dependencies.rb:296:in `block in load'
bundler/gems/rails-a8876760041f/activesupport/lib/active_support/dependencies.rb:268:in `load_dependency'
bundler/gems/rails-a8876760041f/activesupport/lib/active_support/dependencies.rb:296:in `load'
gems/draper-2.1.0/lib/draper/railtie.rb:67:in `block (2 levels) in <class:Railtie>'
gems/draper-2.1.0/lib/draper/railtie.rb:67:in `each'
gems/draper-2.1.0/lib/draper/railtie.rb:67:in `block in <class:Railtie>'
```

Which is an issue with Draper: https://github.com/drapergem/draper/issues/681

That's as far as I've gotten. I just wanted to record my progress here. Once I actually have the test suite running without exceptions, I'll open a PR.

**Comment by seanlinsley** [comment]
Also, there are a lot of deprecation warnings:
```
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from <class:RouteSet> at gems/devise-3.5.2/lib/devise/rails/routes.rb:27)
DEPRECATION WARNING: before_filter is deprecated and will be removed in Rails 5.1. Use before_action instead. (called from block in setup_action_controller at gems/draper-2.1.0/lib/draper.rb:36)
DEPRECATION WARNING: before_filter is deprecated and will be removed in Rails 5.1. Use before_action instead. (called from block (3 levels) in <class:Engine> at gems/turbolinks-2.5.3/lib/turbolinks.rb:14)
DEPRECATION WARNING: after_filter is deprecated and will be removed in Rails 5.1. Use after_action instead. (called from block (3 levels) in <class:Engine> at gems/turbolinks-2.5.3/lib/turbolinks.rb:15)
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from included at gems/turbolinks-2.5.3/lib/turbolinks/xhr_url_for.rb:7)


DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from <class:RouteSet> at gems/devise-3.5.2/lib/devise/rails/routes.rb:27)
DEPRECATION WARNING: before_filter is deprecated and will be removed in Rails 5.1. Use before_action instead. (called from block in setup_action_controller at gems/draper-2.1.0/lib/draper.rb:36)
DEPRECATION WARNING: before_filter is deprecated and will be removed in Rails 5.1. Use before_action instead. (called from block (3 levels) in <class:Engine> at gems/turbolinks-2.5.3/lib/turbolinks.rb:14)
DEPRECATION WARNING: after_filter is deprecated and will be removed in Rails 5.1. Use after_action instead. (called from block (3 levels) in <class:Engine> at gems/turbolinks-2.5.3/lib/turbolinks.rb:15)
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from included at gems/turbolinks-2.5.3/lib/turbolinks/xhr_url_for.rb:7)
DEPRECATION WARNING: prepend_before_filter is deprecated and will be removed in Rails 5.1. Use prepend_before_action instead. (called from <class:DeviseController> at gems/devise-3.5.2/app/controllers/devise_controller.rb:11)
DEPRECATION WARNING: prepend_before_filter is deprecated and will be removed in Rails 5.1. Use prepend_before_action instead. (called from <class:SessionsController> at gems/devise-3.5.2/app/controllers/devise/sessions_controller.rb:2)
DEPRECATION WARNING: prepend_before_filter is deprecated and will be removed in Rails 5.1. Use prepend_before_action instead. (called from <class:SessionsController> at gems/devise-3.5.2/app/controllers/devise/sessions_controller.rb:3)
DEPRECATION WARNING: prepend_before_filter is deprecated and will be removed in Rails 5.1. Use prepend_before_action instead. (called from <class:SessionsController> at gems/devise-3.5.2/app/controllers/devise/sessions_controller.rb:4)
DEPRECATION WARNING: prepend_before_filter is deprecated and will be removed in Rails 5.1. Use prepend_before_action instead. (called from <class:SessionsController> at gems/devise-3.5.2/app/controllers/devise/sessions_controller.rb:5)
DEPRECATION WARNING: prepend_before_filter is deprecated and will be removed in Rails 5.1. Use prepend_before_action instead. (called from <class:PasswordsController> at gems/devise-3.5.2/app/controllers/devise/passwords_controller.rb:2)
DEPRECATION WARNING: append_before_filter is deprecated and will be removed in Rails 5.1. Use append_before_action instead. (called from <class:PasswordsController> at gems/devise-3.5.2/app/controllers/devise/passwords_controller.rb:4)
DEPRECATION WARNING: prepend_before_filter is deprecated and will be removed in Rails 5.1. Use prepend_before_action instead. (called from <class:UnlocksController> at gems/devise-3.5.2/app/controllers/devise/unlocks_controller.rb:2)
DEPRECATION WARNING: prepend_before_filter is deprecated and will be removed in Rails 5.1. Use prepend_before_action instead. (called from <class:RegistrationsController> at gems/devise-3.5.2/app/controllers/devise/registrations_controller.rb:2)
DEPRECATION WARNING: prepend_before_filter is deprecated and will be removed in Rails 5.1. Use prepend_before_action instead. (called from <class:RegistrationsController> at gems/devise-3.5.2/app/controllers/devise/registrations_controller.rb:3)
```

**Comment by seanlinsley** [comment]
Given the deprecation warnings suggesting the use of `Module#prepend`, we won't be able to keep Ruby 1.9 and Rails 3.2 support going for much longer. Which is fine with me.

We're also probably going to release 1.0.0 before Rails 5 is ready, so this won't affect 1.0.0.

**Comment by timoschilling** [comment]
We can use our own `alias_method_chain` as a backport for a while.

**Comment by marcomd** [comment]
Hello,

i'm testing the current master with rails5 beta1, ruby 2.2.3 mingw x64

devise was updated so we can use the master branch:
gem "devise", :github => 'plataformatec/devise', :branch => 'master'

Starting a new app I have this error:

    Formtastic::UnknownInputError at /admin/login
    Unable to find input class Input

**Comment by marcomd** [comment]
```
Started GET "/admin/login" for ::1 at 2016-01-05 09:58:01 +0100
Processing by ActiveAdmin::Devise::SessionsController#new as HTML
  Rendered C:/ruby/ruby-223x64p173/lib/ruby/gems/2.2.0/bundler/gems/activeadmin-ab1d8d1a4a1a/app/views/active_admin/devise/sessions/new.html.erb within layouts/active_admin_logged_out (122.0ms)
Completed 500 Internal Server Error in 291ms (ActiveRecord: 0.0ms)

Formtastic::UnknownInputError - Unable to find input class Input:
  formtastic (3.1.3) lib/formtastic/helpers/input_helper.rb:332:in `rescue in namespaced_input_class'
```

**Comment by seanlinsley** [comment]
@marcomd as I commented on your Formtastic ticket, this is a problem with Devise: https://github.com/plataformatec/devise/issues/3880

**Comment by jtomaszewski** [comment]
Just to let you know guys, following setup makes activeadmin work on rails 5 (at least in my app):

```
# Gemfile
# Wait for https://github.com/activeadmin/activeadmin/pull/4145 to be merged in
gem 'activeadmin', github: 'tjgrathwell/activeadmin', branch: 'rails5'
# Wait for https://github.com/justinfrench/formtastic/pull/1184 to be merged in
gem 'formtastic', github: 'jtomaszewski/formtastic', branch: 'patch-1' 
```

**Comment by marcomd** [comment]
Thank you.
Formtastic was merged so you can use the master.
Unfortunately, i cannot test it on windows due to coffee-script-source https://github.com/jashkenas/coffeescript/issues/4158

**Comment by arogachev** [comment]
Following gem set:

```
# Backend
gem 'activeadmin', github: 'activeadmin/activeadmin', branch: 'rails-5-rspec'

# Authentication
# Master branch is added for Rails 5 support
# https://github.com/plataformatec/devise/pull/3714
gem 'devise', github: 'plataformatec/devise', branch: 'master'

# FormBuilder DSL
gem 'formtastic', github: 'justinfrench/formtastic', branch: 'master'
```

gives an error:

```
/usr/local/rvm/gems/ruby-2.2.2@my_site/bundler/gems/activeadmin-599025964c95/lib/
active_admin/dependency.rb:79:in `match!': You provided devise 4.0.0.rc1 but we
need: ~> 3.2. (ActiveAdmin::DependencyError)
        from /usr/local/rvm/gems/ruby-2.2.2@my_site/bundler/gems/activeadmin-5990
25964c95/lib/active_admin/dependency.rb:48:in `method_missing'
```

See discussion in https://github.com/plataformatec/devise/issues/3880 for more details.

**Comment by devilankur18** [comment]
@arogachev +1

**Comment by micred** [comment]
same here.
Please copy Gemfile.lock of a working Rails 5 setup, so we can specify tre same revisions of Devise + Formtastic + ActiveAdmin to make it work until all stuff will be merged to master.

**Comment by arogachev** [comment]
@micred Here it is.

`Gemfile` contents:

```ruby
# Backend
gem 'activeadmin', github: 'activeadmin/activeadmin', branch: 'rails-5-rspec'

# Authentication
# Master branch is added for Rails 5 support
# https://github.com/plataformatec/devise/pull/3714
gem 'devise', github: 'plataformatec/devise', branch: 'master'
```

`Gemfile.lock` contents:

```
GIT
  remote: git://github.com/activeadmin/activeadmin.git
  revision: 599025964c95d1caaad012c34bd5fb4074729152
  branch: rails-5-rspec
  specs:
    activeadmin (1.0.0.pre2)
      arbre (~> 1.0, >= 1.0.2)
      bourbon
      coffee-rails
      formtastic (~> 3.1)
      formtastic_i18n
      inherited_resources (~> 1.6)
      jquery-rails
      jquery-ui-rails
      kaminari (~> 0.15)
      rails (>= 3.2, < 5.0)
      ransack (~> 1.3)
      sass-rails

GIT
  remote: git://github.com/plataformatec/devise.git
  revision: 1d77099861e2082a6bf5ef9c3a422c2ec315adb2
  branch: master
  specs:
    devise (3.5.3)
      bcrypt (~> 3.0)
      orm_adapter (~> 0.1)
      railties (>= 4.1.0, < 5.1)
      responders
      warden (~> 1.2.3)
```

Note that I didn't explicitly specify Formtastic, so it was downloaded as a dependency of Active Admin. But afterwards when I ran into problem with visible password, I tried to install latest Formtastic on master branch and it worked OK.

But just in case, version of Formtastic before update was `v3.1.3` (I looked in the according rvm gemset folder).

**Comment by micred** [comment]
Great, that devise revision works since it was still marked as 3.5.3 in place of 4.0.0.rc1 as in the current master.

So workaround at the moment is using:

    gem 'devise', :github => 'plataformatec/devise', :ref => '1d77099861e2082a6bf5ef9c3a422c2ec315adb2'
    gem 'formtastic', github: 'justinfrench/formtastic', :branch => 'master' 
    gem 'activeadmin', github: 'activeadmin/activeadmin', :branch => 'master'

**Comment by hmistry** [comment]
Thanks to all of you for clues in solving this issue with Rails 5.0.0.beta2 and rspec. Here's my final work around. In addition, I modified activeadmin's `dependency.rb` to allow `devise v4.0.0.rc1`.

Is draper not being maintained anymore?

```
gem 'activeadmin', github: 'activeadmin/activeadmin', branch: 'rails-5-rspec'
gem 'draper', github: 'versify/draper', branch: 'rails-5'
gem 'pundit'
gem 'activemodel-serializers-xml', github: 'rails/activemodel-serializers-xml'
gem 'formtastic', github: 'justinfrench/formtastic', branch: 'master'

```

**Comment by seanlinsley** [comment]
I have the test suite passing on #4254 if anyone would like to give it a try.

**Comment by marcomd** [comment]
It works!
On windows there is only a problem with https://github.com/codahale/bcrypt-ruby/issues/128 but it is solvable

**Comment by hmistry** [comment]
This did not work for me. A couple things:

1. I noticed the generated scaffold controllers now inherit from `InheritedResources::Base` instead of `ApplicationController`
2. The version of `formtastic` you're supporting is causing the app to break upon sign in. Error was "Unable to find input class Input" on display login form.

This was a new app with ruby 2.3, rails 5.0.0.beta2, devise 4.0.0.rc1.

**Comment by arogachev** [comment]
@hmistry Have you tried to install latest version of Formtastic? As for second error, you can use this [workaround](http://stackoverflow.com/questions/34529098/formtasticunknowninputerror-in-activeadmindevisesessionsnew) before it will be finally fixed.

**Comment by hmistry** [comment]
@arogachev Yes I got things working with the master branch of Formtastic as I stated on this thread 6 days ago. My attempt & comment yesterday was only to help user test @seanlinsley solution on #4254 which I presume should have working dependency gems and working code.

**Comment by marcomd** [comment]
@hmistry
1. I always use activeadmin from my generator in which i already did that, so i can't check it but for me it's a welcome.
2. I used devise-4.0.0.rc1 and formtastic from master 16d94ce8799b, i never get that error

**Comment by seanlinsley** [comment]
#4254 has been merged, so if you're relying on the `rails-5-rspec` branch be sure to switch over to `master`

**Comment by jstnn** [comment]
i've followed this guide to make it work with rails5: https://rrott.com/blog/ror/rails-5-api-with-activeadmin-integration.html

**Comment by rhymes** [comment]
Any news? Rails 5 is probably going to be out in a few weeks http://weblog.rubyonrails.org/2016/5/6/this-week-in-rails-railsconf-recap-rails-5-0-rc-1-is-out/

**Comment by hmistry** [comment]
Update:

I just updated my Rails app from 5.0.0.beta2 to 5.0.0.rc1. I used ActiveAdmin master and only had force Kiminari version in the Gemfile.

```
gem 'activeadmin', github: 'activeadmin/activeadmin' 
gem 'kaminari', github: "amatsuda/kaminari", branch: '0-17-stable' # has fix for activeadmin error
```

**Comment by EtienneDepaulis** [comment]
Hey guys, how have you managed to install AA + Rails 5 rc1 ?

Here is my Gemfile:
```
gem 'rails', '5.0.0.rc1'
gem 'activeadmin', github: "activeadmin/activeadmin"
```

and because of inherited_resources, I keep getting this message when trying to `bundle update rails`:
```
activeadmin was resolved to 1.0.0.pre2, which depends on
      inherited_resources (~> 1.6) was resolved to 1.6.0, which depends on
        railties (< 5, >= 3.2)
rails (= 5.0.0.rc1) was resolved to 5.0.0.rc1, which depends on
      railties (= 5.0.0.rc1)
```

Therefore I don't understand how the current master branch can be installed with `5.0.0.rc1` :/
Thanks for your help

**Comment by micred** [comment]
I think you should do `bundle update activeadmin` before `bundle update rails`

**Comment by EtienneDepaulis** [comment]
Thats what I did.
I have this in my Gemfile.lock:

```
GIT
  remote: git://github.com/activeadmin/activeadmin.git
  revision: e72c62b01397b17123e1b71ef706c5d8e4ffbb2f
  specs:
    activeadmin (1.0.0.pre2)
      arbre (~> 1.0, >= 1.0.2)
      bourbon
      coffee-rails
      formtastic (~> 3.1)
      formtastic_i18n
      inherited_resources (~> 1.6)
      ...
```

e72c62b01397b17123e1b71ef706c5d8e4ffbb2f is the latest commit on the `master` branch.

**Comment by micred** [comment]
I use it without problems with:
```
gem 'devise', :github => 'plataformatec/devise'
gem 'devise-i18n'

gem 'formtastic', github: 'justinfrench/formtastic', branch: 'master' 

gem 'activeadmin', github: 'activeadmin/activeadmin', branch: 'master'

gem 'ransack', github: 'activerecord-hackery/ransack', branch: 'master'

gem 'kaminari', github: 'amatsuda/kaminari', branch: '0-17-stable'
gem 'kaminari-i18n'
```

**Comment by EtienneDepaulis** [comment]
What revision are you using on AA ?

AA latest revision currently requires `inherited_resources` which forbid using rails 5:
```
s.add_dependency("railties", ">= 3.2", "< 5")
```
source: https://github.com/josevalim/inherited_resources/blob/master/inherited_resources.gemspec

Do you have any mention to `inherited_resources` in your `Gemfile.lock` ?

Thanks for your help

**Comment by machty** [comment]
Can someone please explain how they are able to get around the fact that `active_admin` needs `inherited_resources ~> 1.6` but `inherited_resources` explicitly disallows Rails 5?

**Comment by sandfoxme** [comment]
@machty That's about bundler version matcher. pre-released version are slightly "lower". (5.0.0.rc1 < 5)
So it works for now and when Rails 5 is released it will break

**Comment by hmistry** [comment]
@machty I don't know what to tell you but it just worked for me. I'm using Rails 5.0.0.rc1, AA `e72c62b`, and Kaminari branch 0-17-stable (this was important to get AA installed). I see `inherited_resources ~> 1.6` in my Gemfile.lock but it didn't prevent my install.

**Comment by mcolyer** [comment]
I'm currently using `1.0.0.pre2` with rails `5.0.0rc1` and I noticed that filters don't seem to be working (ie the page loads but none of the objects returned are filtered), is this a known issue?

**Comment by alexperezpaya** [comment]
#4490

**Comment by t-anjan** [comment]
Rails 5.0 has been released. As @sandfoxme said, now, ActiveAdmin is broken. Any updates?

**Comment by timoschilling** [comment]
@t-anjan not ActiveAdmin is broken, the Problem is inherited_resources and has_scope.

**Comment by mattclar** [comment]
@timoschilling this may be true but for inherited_resources it is no longer being developed and is unlikely to see the necessary updates to rails 5

**Comment by timoschilling** [comment]
@mattclar inherited_resources will bring a new release shortly

**Comment by ro4tub** [comment]
It seems like there is something wrong with the database_cleaner gem.

```
bundle install
Updating git://github.com/pschambacher/database_cleaner.git
fatal: ambiguous argument '8dd9fa4': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
Git error: command `git rev-parse 8dd9fa4` in directory /Users/oscar/.rvm/gems/ruby-2.3.0/cache/bundler/git/database_cleaner-58e14124eeebbda50a43c2679c93834d78a02266 has failed.
If this error persists you could try removing the cache directory '/Users/oscar/.rvm/gems/ruby-2.3.0/cache/bundler/git/database_cleaner-58e14124eeebbda50a43c2679c93834d78a02266'
```

**Comment by rhymes** [comment]
@ro4tub it seems more a git problem, anyway you should report it on database_cleaner github, not an activeadmin's :D

Also, you're not using the official DatabaseCleaner git repository, which is https://github.com/DatabaseCleaner/database_cleaner

BTW do as the error message suggests, remove the cache directory and try again.

**Comment by ro4tub** [comment]
@rhymes I followed the instructions on the page https://github.com/activeadmin/activeadmin.
I did as the error message said, the same error.
Also I removed the git version ref in the `gem 'database_cleaner',            github: 'pschambacher/database_cleaner', branch: 'rails5.0', ref: '8dd9fa4'`, `bundle install` ok, but `rails generate active_admin:install` failed.

**Comment by rhymes** [comment]
@ro4tub you're correct, I was mistaken. I inspected the repositories and the repo/branch combo you're referring to is basically updated as of September 2015 with an additional commit of March 2016. Also I can't find that ref 8dd9fa4. I suspect something changed in that repo and it's not up to date (nor is ActiveAdmin's README apparently). It's a lot of commits behind the official version also.

If you really need DatabaseCleaner try using the official github version, I do see there's an issue with AR 5.0 here https://github.com/DatabaseCleaner/database_cleaner/issues/445 but it seems there's a workaround

If it does work open an issue here to update the README, if it doesn't comment DatabaseCleaner's issue 445 and get help from them and then maybe report here how you solved it so one of the devs can do the necessary updates (either to the doc or the code)

**Comment by ro4tub** [comment]
@rhymes  cool, it resolved. Thank you.
I created a README issue.

**Comment by seanlinsley** [comment]
I just opened #4522 to update the version requirements. I just released Draper 3.0.0.pre1, and a number of other gems have pushed Rails 5 compatible releases to Rubygems.

These are the only version requirements that need to be manually specified:
```ruby
gem 'inherited_resources', github: 'activeadmin/inherited_resources'
gem 'ransack',             github: 'activerecord-hackery/ransack'
gem 'draper',              '> 3.x'
```
`bundle update activeadmin` will correctly resolve the others.

**Comment by ManojDatt** [comment]
I was getting the fallowing error while bundle activeadmin:
```
Bundler could not find compatible versions for gem "actionpack":
  In snapshot (Gemfile.lock):
    actionpack (= 5.0.0)

  In Gemfile:
    activeadmin was resolved to 1.0.0.pre4, which depends on
      inherited_resources (~> 1.6) was resolved to 1.6.0, which depends on
        actionpack (< 5, >= 3.2)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      railties (= 5.0.0) was resolved to 5.0.0, which depends on
        actionpack (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      railties (= 5.0.0) was resolved to 5.0.0, which depends on
        actionpack (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      railties (= 5.0.0) was resolved to 5.0.0, which depends on
        actionpack (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      railties (= 5.0.0) was resolved to 5.0.0, which depends on
        actionpack (= 5.0.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      sprockets-rails (>= 2.0.0) was resolved to 3.1.1, which depends on
        actionpack (>= 4.0)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

-------------------------------------------------------------------------------------------------------------------------------
_

> To solve this error and install activeadmin i uses the fallowing gem configuration in gem-file and it works well::

_

```
**gem 'devise'
gem 'activeadmin', github: 'activeadmin/activeadmin'
gem 'inherited_resources', github: 'activeadmin/inherited_resources'**
```

**Comment by timoschilling** [comment]
I'm closing this issue, while we have a general support for Rails 5. If there are new issues, please read the Rails 5 section of the readme, if that don't help create a new issue.
